tests/src/test/java/alluxio/shell/command/TestCommandTest.java

```diff
@@ -0,0 +1,184 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.shell.command;
+
+import alluxio.AlluxioURI;
+import alluxio.client.FileSystemTestUtils;
+import alluxio.client.WriteType;
+import alluxio.exception.AlluxioException;
+import alluxio.shell.AbstractAlluxioShellTest;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.io.IOException;
+
+/**
+ * Tests for test command.
+ */
+public final class TestCommandTest extends AbstractAlluxioShellTest {
+  @Test
+  public void testPathIsDirectoryWhenPathNotExist() throws IOException {
+    mFsShell.run("test", "-d", "/testPath");
```

Comment:
same here, by returning 1/0 instead of printing them, we should have:
```java
int ret = mFsShell.run("test", "-d", "/testPath");
Assert.assertEquals(1, ret)
```